internal/topology/check/compatibility_test.go

```diff
@@ -706,19 +706,19 @@ func TestMachineDeploymentClassesAreCompatible(t *testing.T) {
 		APIVersion: "group.test.io/foo",
 		Kind:       "barTemplate",
 		Name:       "baz",
-		Namespace:  "default",
+		Namespace:  "other",
```

Comment:
Let's keep this at default as well (otherwise we don't verify that it's okay to change namespace)

Same in l.855